force_wfmanager/ui/review/tests/test_plot.py

```diff
@@ -230,7 +230,7 @@ def test_select_plot_axis(self):
             mock_update.assert_called()
 
         self.plot._update_plot()
-        self.assertEqual((1.8, 4.2, 3.0, 3.0), self.plot._get_plot_range())
+        self.assertEqual((1.8, 4.2, 2.5, 3.5), self.plot._get_plot_range())
```

Comment:
Just checking - presumably this test was incorrect before? 

It looks like the updated version should be what we expect to return

Reply:
if you do print(data) at the start of calculate_axis_bounds(data), you'll find
[2. 4.]   > 1.8, 4.2
[3. 3.]  > 2.5, 3.5  (not 3.0, 3.0)

So yes the original assert was wrong.

Reply:
I agree - looking closer I think this change also provides the unit test that I requested